vue-ui/src/router/index.js

```diff
@@ -10,21 +29,21 @@ const router = createRouter({
       component: HomeView,
     },
     {
-      path: '/about',
-      name: 'about',
-      // route level code-splitting
-      // this generates a separate chunk (About.[hash].js) for this route
-      // which is lazy-loaded when the route is visited.
-      component: () => import('../views/AboutView.vue'),
-    },
-    {
-      path: '/test-chart',
-      name: 'test',
+      path: '/south-bird-island-chart',
+      name: 'southBirdIslandChart',
+
       // route level code-splitting
       // this generates a separate chunk (About.[hash].js) for this route
       // which is lazy-loaded when the route is visited.
-      component: () => import('../views/TestChartView.vue'),
-    },
+      component: () => {
+        // Check the screen size and load the appropriate component
+        if (window.innerWidth <= 480) {
```

Comment:
This is the part when it will get complicated once scaling up.
In the future, I suggest using `tailwindcss` for responsive css instead of raw CSS.
With tailwind you don't need to create 2 views for each page, just one page itself and very minimal css code.

Once again, no change needed now, you can `Resolve Conversation` after you read it as acknowledgement of my comment.